src/Resolver/Resolver.php

```diff
@@ -12,7 +12,7 @@
  */
 final class Resolver implements ResolverInterface
 {
-    private $executor;
+    protected $executor;
```

Comment:
Hm? This is a `final` class anyway, so how much is gained by changing its visibility?